docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -72,6 +73,38 @@ To hide the username, we need to mask it.
         .sentryReplayMask()
     }
     .background(.blue)
-    .sentryReplayUnmask() 
+    .sentryReplayUnmask()
+  }
+```
+
+## Debugging Session Replay masking
+
+To see how elements are being masked, enable the masking preview. It will display a semi-transparent red overlay on top of the masked elements.
```

Comment:
```suggestion
To see how elements are being masked, enable the masking preview. It will display a semi-transparent overlay on top of the masked elements.
```